app/page.tsx

```diff
@@ -53,43 +53,60 @@ async function getMsgExpRecipeCollection(): Promise<NimbusRecipeCollection> {
   await recipeCollection.fetchRecipes()
```

Comment:
Instead of calling fetchRecipes twice, please hoist this call and the rollouts one to the Dashboard component, create a collection for everything and fetch once there, then make both getMsgExpRecipeCollection and getMessageRolloutRecipeCollection take a NimbusRecipeCollection as a parameter.